src/plugins/intel_gpu/src/kernel_selector/CMakeLists.txt

```diff
@@ -75,7 +75,7 @@ if(COMMAND add_cpplint_target)
   add_cpplint_target(${TARGET_NAME}_cpplint FOR_TARGETS ${TARGET_NAME})
 endif()
 
-target_link_libraries(${TARGET_NAME} PUBLIC OpenCL::OpenCL openvino::runtime)
+target_link_libraries(${TARGET_NAME} PUBLIC OpenCL::OpenCL openvino::runtime PRIVATE openvino::runtime::dev)
```

Comment:
Needed for visibility of ov_ops/swiglu.hpp in 
https://github.com/openvinotoolkit/openvino/blob/7566bc94c58a501389647b4cc4d7c21df311fa63/src/plugins/intel_gpu/src/kernel_selector/kernels/swiglu/swiglu_kernel_base.h#L9